build/Dockerfile

```diff
@@ -116,8 +114,8 @@ ENV CXX /x-tools/x86_64-unknown-linux-gnu/bin/x86_64-unknown-linux-gnu-g++
 # BEGIN https://github.com/docker-library/golang/blob/master/1.8/alpine/Dockerfile
 
 COPY parallelbuilds-go1.8.patch /
-RUN curl -fsSL https://golang.org/dl/go1.8.src.tar.gz -o golang.tar.gz \
- && echo '406865f587b44be7092f206d73fc1de252600b79b3cacc587b74b5ef5c623596  golang.tar.gz' | sha256sum -c - \
+RUN curl -fsSL https://storage.googleapis.com/golang/go1.8.1.src.tar.gz -o golang.tar.gz \
```

Comment:
I don't know, ask the go team. All the links on https://golang.org/dl/ point here now.